### PR TITLE
add packages to enable native fencing for microsoft-azure

### PR DESCRIPTION
**Pull request by yeoldegrove**
Like commented in here https://github.com/SUSE/ha-sap-terraform-deployments/pull/714#discussion_r659591573 ... this is the better place to add packages for the microsoft-azure native fencing.

### CODE REVIEW

**Comment by yeoldegrove** [comment]
@arbulu89 feel free to review and merge

File: habootstrap-formula.changes [hunk 1]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Mon Jun 28 17:24:06 UTC 2021 - Eike Waldt <waldt@b1-systems.de>
+
+- Version bump 0.4.3
+  *  add packages to enable native fencing for microsoft-azure
```
Comment (arbulu89) [inline habootstrap-formula.changes]:
```suggestion
  * Add packages to enable native fencing for microsoft-azure
```

Comment (arbulu89) [reply habootstrap-formula.changes]: I'm fixing this changelog entry to be more consistent

**Review by arbulu89 — APPROVED** [review]
@yeoldegrove Thanks

**Pull request merged**